native_client/definitions.mk

```diff
@@ -55,7 +55,7 @@ PYTHON_PLATFORM_NAME ?= --plat-name manylinux1_x86_64
 endif
 endif
 
-ifeq ($(TARGET),host-win)
+ifeq ($(findstring _NT,$(OS)),_NT)
```

Comment:
I'd prefet we keep `TARGET=host-win` and change the way we detect and set the variable

Reply:
Why?

Reply:
It felt "nicer", but at the same time, I see we already have it `findstring` at several places.